pkg/apis/samples/v1alpha1/conversion.go

```diff
@@ -0,0 +1,56 @@
+package v1alpha1
+
+import (
+	conversion "k8s.io/apimachinery/pkg/conversion"
+	duckv1beta1 "knative.dev/pkg/apis/duck/v1beta1"
+	"knative.dev/sample-controller/pkg/apis/samples"
+)
+
+func Convert_v1alpha1_AddressableServiceSpec_To_samples_AddressableServiceSpec(
```

Comment:
How does this relate to the zz_generated.conversion.go stuff?

Reply:
`conversion-gen` can't produce conversions for all types so it defers to the developer and it expects these functions to have a certain name and inputs.